src/services/subject.js

```
@@ -17,15 +20,23 @@ const getSubjectById = async (id) => {
 }
 
 const getSubjectsByCategoryId = async (categoryId) => {
-  const subjects = await Subject.find({ categoryId }, 'name')
-  return subjects.map((subject) => {
-    return {
-      name: subject.name,
-      _id: subject._id
-    }
-  })
+  let query = {}
```

ESLint

[error] Expected linebreaks to be 'CRLF' but found 'LF'.

(linebreak-style)